author.tf

```diff
@@ -231,6 +231,53 @@ module "author-survey-runner-static" {
   memory_units               = "512"
 }
 
+module "author-survey-launcher" {
+  source                   = "github.com/ONSdigital/eq-ecs-deploy?ref=v4.1"
+  env                      = "${var.env}"
+  aws_account_id           = "${var.aws_account_id}"
+  aws_assume_role_arn      = "${var.aws_assume_role_arn}"
+  vpc_id                   = "${module.author-vpc.vpc_id}"
+  dns_zone_name            = "${var.dns_zone_name}"
+  ecs_cluster_name         = "${module.author-eq-ecs.ecs_cluster_name}"
+  aws_alb_arn              = "${module.author-eq-ecs.aws_external_alb_arn}"
+  aws_alb_listener_arn     = "${module.author-eq-ecs.aws_external_alb_listener_arn}"
+  service_name             = "surveys-launch"
+  listener_rule_priority   = 700
+  docker_registry          = "${var.author_registry}"
+  container_name           = "go-launch-a-survey"
+  container_port           = 8000
+  healthcheck_path         = "/status"
+  container_tag            = "${var.survey_launcher_tag}"
+  application_min_tasks    = "${var.survey_launcher_min_tasks}"
+  ecs_subnet_ids           = "${module.author-eq-ecs.ecs_subnet_ids}"
+  ecs_alb_security_group   = ["${module.author-eq-ecs.ecs_alb_security_group}"]
+  launch_type              = "FARGATE"
+  slack_alert_sns_arn      = "${module.eq-alerting.slack_alert_sns_arn}"
+
+  container_environment_variables = <<EOF
+      {
+        "name": "SURVEY_RUNNER_URL",
+        "value": "${module.author-survey-runner.service_address}"
```

Comment:
When deploying this to pre-prod and prod, we probably want to point this at survey runner's pre-prod environment rather than our own. This will allow for testing end to end (including downstream submission) from the registry.